features/step_definitions/admin_acas_steps.rb

```diff
@@ -66,3 +65,58 @@
 Then(/^I can download the contents of the acas document$/) do
   expect(admin_pages.acas_search_results_page).to have_download_link_for(@certificate)
 end
+
+When(/^an ET Administrator with full access can view successful Acas Certificate log$/) do
+  admin_username = ::EtFullSystem::Test::Configuration.admin_username
+  admin_password = ::EtFullSystem::Test::Configuration.admin_password
+  admin_pages.logout_page.load
+  admin_pages.dashboard_page.load
+  if admin_pages.login_page.displayed?
+    admin_pages.login_page.login(email: admin_username, password: admin_password)
+    raise "Could not login to admin with username '#{admin_username}' and password '#{admin_password}'" unless admin_pages.dashboard_page.displayed?
+  end
+
+  if ENV['ENVIRONMENT'] == 'local'
```

Comment:
Cant this environment variable be empty ? Should it then not assume local ? I think I got it from the config class before as this had the default value